src/Github/CreateReleaseTextThroughChangelog.php

```diff
@@ -56,6 +59,16 @@ public function __invoke(
         return $text;
     }
 
+    public function canCreateReleaseText(
+        Milestone $milestone,
+        RepositoryName $repositoryName,
+        SemVerVersion $semVerVersion,
+        BranchName $sourceBranch,
+        string $repositoryDirectory
```

Comment:
Not important in this patch, so feel free to dismiss (or creating a new issue for me to solve later), but I think we're repeating `$repositoryDirectory` **a lot** across this codebase.

A value type would prevent us from passing a random `non-empty-string` in here.

Reply:
I'd be willing to add that as part of this patch if you'd like; I rely on it **heavily**, as I need to work with the filesystem.

Reply:
Let's defer: I'll create an issue for this. Important right now is testing (which I'm doing on my little frankenstein repo @ https://github.com/Ocramius/just-trying-automated-releases ) and then I think we should just ship this :)